packages/components/src/Checkbox/Checkbox.test.tsx

```diff
@@ -71,31 +72,29 @@ test('allows to render without label', () => {
 
 test('supports read only state', () => {
   render(
-    <Checkbox data-testid="checkbox" readOnly defaultChecked>
+    <MyCheckbox data-testid="checkbox" readOnly defaultChecked>
       Read Only
-    </Checkbox>
+    </MyCheckbox>
   );
 
-  const checkbox = screen.getByTestId<HTMLInputElement>('checkbox');
+  const checkbox = screen.getAllByTestId<HTMLInputElement>('checkbox')[1];
   const component = screen.getByText('Read Only');
 
   fireEvent.click(component);
   expect(checkbox.checked).toBeTruthy();
 });
 
 test('check if all slot class names are applied correctly', () => {
-  render(<Checkbox data-testid="checkbox">With Label</Checkbox>);
+  render(<MyCheckbox data-testid="checkbox">With Label</MyCheckbox>);
 
   const label = screen.getByText('With Label');
 
   expect(label.parentElement?.className).toMatchInlineSnapshot(
-    `"group/checkbox relative flex items-center gap-[1ch]"`
+    `"group/checkbox flex items-center gap-[0.5rem] cursor-pointer data-[disabled]:cursor-not-allowed"`
   );
   expect(
     (label.parentElement?.childNodes[0] as HTMLElement).className
-  ).toMatchInlineSnapshot(
-    `"z-1 absolute left-0 top-0 h-full w-full cursor-pointer opacity-[0.0001] group-disabled/checkbox:cursor-not-allowed"`
-  );
+  ).toMatchInlineSnapshot(`""`);
```

Comment:
🤔

Reply:
i guess no need for this test since we have the input already injected inside the `Checkbox` component from `rac`